src/runtime/node/module.ts

```diff
@@ -130,28 +135,30 @@ export const _resolveFilename = /*@__PURE__*/ notImplemented(
 export const _resolveLookupPaths = /*@__PURE__*/ notImplemented(
   "module._resolveLookupPaths",
 );
+export const _stat = /*@__PURE__*/ notImplemented("module._stat");
+export const _readPackage = /*@__PURE__*/ notImplemented("module._readPackage");
 
 export const _pathCache = Object.create(null);
 export const globalPaths = ["node_modules"];
 
-export class _Module implements NodeJS.Module {
-  require: NodeJS.Require;
-  id: string;
-  filename: string;
-  exports = Object.create(null);
-  parent = undefined;
-  loaded = true;
-  children = [];
-  isPreloading: boolean = false;
-  path: string = "/";
-  paths: string[] = [];
-
-  constructor(id = "index.js") {
-    this.id = id;
-    this.filename = id;
-    this.require = createRequire(id);
-  }
-}
+// export class _Module implements NodeJS.Module {
```

Comment:
@vicb i checked cloudflare preset also it seems it doesn't need it. by any chance do you remember why we initially added this export? (if no use we can cleanup code)

Reply:
It was introduced in [June](https://github.com/unjs/unenv/pull/269/) before I joined cf.

Maybe @IgorMinar @petebacondarwin or @dario-piotrowicz can help?